mungegithub/mungers/publish_scripts/publish_template.sh

```diff
@@ -59,27 +60,26 @@ readonly SRC_BRANCH DST_BRANCH DEPS KUBERNETES_REMOTE IS_LIBRARY
 SCRIPT_DIR=$(dirname "${BASH_SOURCE}")
 source "${SCRIPT_DIR}"/util.sh
 
-git fetch origin
-if [ "$(git rev-parse --abbrev-ref HEAD)" = "${DST_BRANCH}" ]; then
-    git reset --hard origin/"${DST_BRANCH}"
+echo "Fetching from origin."
+git fetch origin --no-tags
```

Comment:
Is `--no-tags` intentional?

Reply:
Yes, the tag PR will depend on it. Here it won't matter yet.